aten/src/ATen/test/quantized_test.cpp

```diff
@@ -20,11 +20,7 @@ TEST(TestQTensor, QuantDequantAPIs) {
   Tensor qr = r.quantize_linear(scale, zero_point);
   ASSERT_EQ(qr.q_scale().to<float>(), scale);
   ASSERT_EQ(qr.q_zero_point().to<int32_t>(), zero_point);
-
-  // TODO: Uncomment when quantizer is ready
-  // auto* quantizer = static_cast<PerTensorAffineQuantizer*>(qr.quantizer());
-  // ASSERT_EQ(quantizer->scale(), scale);
-  // ASSERT_EQ(quantizer->zero_point(), zero_point);
+  ASSERT_TRUE(qr.is_quantized());
 
```

Comment:
Please expand test coverage. Replace Tensor r=at::ones({num_elements}) with a tensor with elements in the range (-1,126.5) and change scale to 0.5. This way we can check if quantize_linear actually does quantization, instead of the current case where there is no quantization error.

Reply:
I have the test of quantized value in next diff